src/mococrw/openssl_lib.h

```diff
@@ -58,6 +59,24 @@ namespace lib
 class OpenSSLLib
 {
 public:
+    static int SSL_ENGINE_free(ENGINE* e) noexcept;
```

Comment:
Here the changes are added at the top. In the C file they are added at the bottom.
Any reason for this?

Reply:
The add_function.py script automatically adds the OpenSSL's API trigger for MoCOCrW.